packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -131,36 +131,36 @@ If locking is not set in an `InnerBlocks` area: the locking of the parent `Inner
 
 If the block is a top level block: the locking of the Custom Post Type is used.
 
-### `renderAppender`
+### `appender`
 
--   **Type:** `Component|false`
--   **Default:** - `undefined`. When `renderAppender` is not specified, the default appender is shown. If a `false` value is provided, no appender is rendered.
+-   **Type:** `Object|false`
```

Comment:
I think it should be `Object|null` now?

Reply:
Ah, yeah! Good catch! Amended here: https://github.com/WordPress/gutenberg/pull/29911/commits/463ab85a5ca460d5e8fbd58dbe4bbd9b0a59ea8e